src/modules/myApp.js

```diff
@@ -0,0 +1,67 @@
+const makeRequest = async (url, method = 'GET', requestData = null) => {
+  const headers = {
+    'Content-Type': 'application/json',
+  };
+
+  const options = {
+    method,
+    headers,
+  };
+
+  if (requestData) {
+    options.body = JSON.stringify(requestData);
+  }
+
+  const response = await fetch(url, options);
+  const data = await response.json();
+
+  if (!response.ok) {
+    throw new Error(data.message || 'Request failed');
+  }
+
+  return data;
+};
+
+const createGame = async (baseUrl, name) => {
+  const url = `${baseUrl}games/`;
+  const requestData = { name };
+  const response = await makeRequest(url, 'POST', requestData);
+  return response.result;
+};
+
+const initializeGameId = async (baseUrl, gameName) => {
+  const gameId = localStorage.getItem(gameName);
+  if (!gameId) {
+    try {
+      const newGameId = await createGame(baseUrl, gameName);
+      localStorage.setItem(gameName, newGameId);
+      return newGameId;
+    } catch (error) {
+      return Promise.reject(error);
+    }
+  } else {
+    return gameId;
+  }
+};
+
+const addScore = async (baseUrl, gameId, user, score) => {
+  const url = `${baseUrl}games/${gameId}/scores/`;
+  const requestData = { user, score };
+  const response = await makeRequest(url, 'POST', requestData);
+  return response.result;
+};
```

Comment:
- [OPTIONAL] As a nice UI, the scores list does NOT need to be refreshed at this point. It should happen when the refresh button is clicked.

Reply:
Okay I'll make the change